src/store/index.tsx

```diff
@@ -1,10 +1,19 @@
 import { configureStore } from '@reduxjs/toolkit';
 import counterReducer from '../features/counter/counterSlice';
-
+import { bookSearchApi } from '@/features/BookSearchPage/api/bookSearchApi';
+import { bestBookGetApi } from '@/features/BookSearchPage/api/bestBookGetApi';
+import bookSearchReducer from '@/features/BookSearchPage/Slice/bookSearchSlice';
 export const store = configureStore({
   reducer: {
+    bookSearch: bookSearchReducer,
+    [bookSearchApi.reducerPath]: bookSearchApi.reducer,
+    [bestBookGetApi.reducerPath]: bestBookGetApi.reducer,
     counter: counterReducer,
   },
+  middleware: (getDefaultMiddleware) =>
+    getDefaultMiddleware()
+      .concat(bookSearchApi.middleware)
+      .concat(bestBookGetApi.middleware), // 미들웨어 추가
```

Comment:
```suggestion
      .concat([bookSearchApi.middleware, bestBookGetApi.middleware])
```
이렇게 배열 형태로 작성하면 미들웨어 추가 시 concat 메소드 추가할 필요없이 미들웨어만 추가할 수 있어서 간결할 듯 합니다!

Reply:
감사합니다 ! 수정 하였습니다.